plugins/techdocs-node/api-report.md

```diff
@@ -24,6 +24,8 @@ export class DirectoryPreparer implements PreparerBase {
     { logger, reader }: PreparerConfig,
   ): DirectoryPreparer;
   prepare(entity: Entity, options?: PreparerOptions): Promise<PreparerResponse>;
+  // (undocumented)
```

Comment:
Could you [inherit](https://api-extractor.com/pages/tsdoc/tag_inheritdoc/) the documentation here?